### DISCUSSION TITLE
How to translate contents?

### DISCUSSION THREAD

**Discussion by PappoYT** [OFFLINE-GmbH/oc-mall-plugin#892] [Q&A] [answered]
Is there a way to translate strings through backend?
At the moment I'm editing the file `/plugins/offline/mall/lang/it/frontend.php` with my Notepad++ but I think it should be more comfortable to do this things from OctoberCMS backend but I cannot find how to do.
If I go in `settings > translate > translate messages` I don't find anything (see attached image).
TY
![OctoberCMS-translate-messages](https://user-images.githubusercontent.com/74958346/147679247-5f8d0643-1e19-4e75-a75f-c7c58ed03399.PNG)

**Comment by tobias-kuendig** [comment, marked as answer]
This is how translations work in October. Plugins provide their own translations. The RainLab.Translate plugin is there to translate theme content. 

You can create a pull request to this repo with your added translations so everybody benefits.

If you want to change existing translations from the plugin just for your usecase you can use this method:
https://octobertricks.com/tricks/override-translations-with-a-plugin

**Reply by mjauvin** [reply]
can't you just override the plugin translations in your app's lang folder as described here?
https://docs.octobercms.com/2.x/plugin/localization.html
